Clinical-Genomics/scout
# Add cnv report side panel#1917

This PR adds the possibility to download CNV reports (fix #1917)

**How to test**:
1. On stage, deploy the branch
2. Check that it is possible to download the report on casehttps://scout-stage.scilifelab.se/cust000/actualgecko
3. Other cases should not have a link to the report

**Expected outcome**:
The functionality should be working!

**Review:**
- [x] tests executed by hassanfa
